types/react-select/lib/components/Menu.d.ts

```diff
@@ -67,9 +67,6 @@ export type MenuProps<OptionType> = CommonProps<OptionType> & {
 export function menuCSS(state: MenuState): React.CSSProperties;
 
 export class Menu<OptionType> extends Component<MenuProps<OptionType>, MenuState> {
-  static contextTypes: {
```

Comment:
Is this removed because it's the wrong type, or because it's no-longer possible to define this static method?  This is based on the source @ https://github.com/JedWatson/react-select/blob/master/src/components/Menu.js#L267

Reply:
It's the wrong type.  Each member of `contextTypes` should be a validator for the type of the context entry, not an actual value of the type.  It would probably work to write `getPortalPlacement: typeof PropTypes.func`, but I didn't see the benefit of keeping this declaration at all.